README.md

```diff
@@ -66,122 +52,175 @@ Once the environment settings has been sourced, you are ready to proceed with th
 
 ### Quick start example
 
-```
-$ python3 fpgaperf.py --toolchain vivado --project oneblink --board arty
+```bash
+python3 fpgaperf.py --toolchain vivado --project oneblink --board arty
 ```
 
 or
 
-```
-$ python3 fpgaperf.py --toolchain vpr --project oneblink --board basys3
+```bash
+python3 fpgaperf.py --toolchain vpr --project oneblink --board basys3
 ```
 
 For example to compare pure Vivado flow and Yosys -> Vivado flow for an xc7z device the following commands can be run:
 
-```
+```bash
 # Yosys -> Vivado
-$ python3 fpgaperf.py --toolchain vivado-yosys --project oneblink --board basys3
+python3 fpgaperf.py --toolchain vivado-yosys --project oneblink --board basys3
 # Pure Vivado
-$ python3 fpgaperf.py --toolchain vivado --project oneblink --board basys3
+python3 fpgaperf.py --toolchain vivado --project oneblink --board basys3
 ```
 
 Use `--help` to see additional parameters for the `fpgaperf.py` script.
 
 Supported toolchains can be queried as follows:
-```
+```bash
 $ python3 fpgaperf.py  --list-toolchains
-Supported toolchains:
-arachne
 icecube2-lse
 icecube2-synpro
 icecube2-yosys
-nextpnr
+nextpnr-ice40
+nextpnr-xilinx
+nextpnr-xilinx-fasm2bels
 radiant-lse
 radiant-synpro
 vivado
 vivado-yosys
 vpr
+vpr-fasm2bels
 ```
 
 You can check if you have the toolchain environments correctly installed as
 follows:
-```
+```bash
 $ python3 fpgaperf.py --check-env --toolchain vpr
 vpr
   yosys: True
   vpr: True
-  icebox_hlc2asc: True
-  icepack: True
-  icetime: True
 ```
 
 Supported projects can be queried as follows:
-```
+```bash
 $ python3 fpgaperf.py  --list-projects
 baselitex
+blinky
+hamsternz-hdmi
 ibex
 murax
 oneblink
 picorv32
 picosoc
-picosoc_hx8kdemo
-picosoc_simpleuart
-picosoc_spimemio
+picosoc-simpleuart
+picosoc-spimemio
 vexriscv
+vexriscv-smp
 ```
 
 ### Exhaustive build
 
 Use `exhaust.py` to automatically test all projects, toolchain and boards supported
 
-```
-$ python3 exhaust.py
+```bash
+python3 exhaust.py
 ```
 
-Its also possible to run a test against a single toolchain and/or project and/or device:
-```
-$ python3 exhaust.py --project blinky --toolchain nextpnr
+Its also possible to run a test against a single toolchain and/or project:
+```bash
+python3 exhaust.py --project blinky --toolchain vpr
 ```
 
-See build directory for output. Note in particular [build/all.json](build/all.json)
+See build directory for output. Note in particular `all.json`.
 
+## Project Structure
 
-## Development
+This section describes the structure of this project to better understand its mechanisms.
 
-### Project
```

Comment:
Moved this section to below the section on Wrappers, and right before the inserting new project instructions.